### PR TITLE
Use bindata-assetfs to bundle assets and templates

### PR DESCRIPTION
**Pull request by alde**
Add go generate to Makefile
Stop copying assets and templates to docker folder

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,9 @@ SRC=$(shell find . -name '*.go')
 all: test
 
 deps:
+	go get github.com/jteeuwen/go-bindata/...
+	go get github.com/elazarl/go-bindata-assetfs/...
+	go generate
```
Comment (keis) [inline Makefile]: does this need to be done before `go get` or could it be added to the eremetic block before `go build`?

Comment (alde) [inline Makefile]: Must be before `go get -t ./...` or the go get fails because there's no .go file in the assets dir

Comment (keis) [inline Makefile]: oh ok. it's a bit like a catch 22 but if it works that's fine I guess.

**Comment by keis** [comment]
[LGTM](http://lgtm.in/i/cOSt1y0Yp)

**Pull request merged**